src/JiYeong/filterJY/components/categoryTilte/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from "react";
+
+import { filterContentsArr } from "../../core/filterContents";
+import { resultContentsArr } from "../../core/resultContents";
+import { St } from "./style";
+
+export default function FilterContents() {
+  const [pickedContents, setPickedContents] = useState<Set<string>>(new Set()); //선택된 태그들을 담은 Set
+  const [tagArr, setTagArr] = useState<string[][]>([]); // 결과 데이터의 태그들을 담은 배열
+
+  // 태그를 선택했을 때 선택된 태그를 담는 배열을 생성
+  const clickContent = (index1: number, index2: number) => {
+    console.log(index1, index2);
+    const clickedText = filterContentsArr[index1].contents[index2];
+    const tempClickedContents = new Set([...pickedContents]);
+    tempClickedContents.has(clickedText)
+      ? tempClickedContents.delete(clickedText)
+      : tempClickedContents.add(clickedText);
+    setPickedContents(tempClickedContents);
+  };
+
+  // 태그 담는 배열이 바뀔 때마다 콘솔 찍어보기
+  useEffect(() => {
+    console.log(pickedContents);
+  }, [pickedContents]);
+
+  // 더미데이터의 콘텐츠에 있는 태그들이 선택한 태그랑 같은지 확인할 수 있는 배열 생성
+  const createTagArr = () => {
+    const tempTagArr: string[][] = [...tagArr];
+    resultContentsArr.map((id) => {
+      const tempText = [...id.gender, ...id.age, ...id.intimacy];
+      tempTagArr.push(tempText);
+      setTagArr([...tempTagArr]);
+      console.log(tempTagArr);
+    });
+  };
+
+  // 위의 배열을 처음 랜더링 할 때 1번만 생성하기!
+  useEffect(() => {
+    createTagArr();
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    createTagArr();
  }, [createTagArr]);
```

린트 warning 무시하지 말아요 !!!~~!